lib/template.html

```diff
@@ -85,11 +85,19 @@
 {% endif %}
 
 {% if site.logo %}
-    {% assign seo_site_logo = site.logo | prepend: seo_url | escape %}
+  {% assign seo_site_logo = site.logo %}
+  {% unless seo_site_logo contains "://" %}
+    {% assign seo_site_logo = seo_site_logo | prepend: seo_url %}
+  {% endunless %}
+  {% assign seo_site_logo = seo_site_logo | escape %}
 {% endif %}
 
 {% if page.image %}
-    {% assign seo_page_image = page.image | prepend: seo_url | escape %}
+  {% assign seo_page_image = page.image %}
+  {% unless seo_page_image contains "://" %}
```

Comment:
contains "//" in case of prefix-free URLs like  `//jekyllrb.com/img/logo-2x.png` ?